tequilapi/endpoints/identities.go

```diff
@@ -124,7 +124,7 @@ func validateRegistrationRequest(regReq identityRegistrationDto) (err error) {
 func validateCreationRequest(createReq *identityCreationDto) (errors *validation.FieldErrorMap) {
 	errors = validation.NewErrorMap()
 	if createReq.Password == nil {
-		errors.ForField("password").AddError("required", "Field is required")
+		//errors.ForField("password").AddError("required", "Field is required")
```

Comment:
Why this line is commented out?

Reply:
This should be disabled for pre-alpha.

Reply:
We can remove it to remove cluster - we can always check git history to revert that if needed.

Reply:
This shouldn't be commented-out, because it already allows passing empty passwords. It just checks that `password` param was passed.
This allows `{"password": ""}`, but forbids `{}`.